_i18n/en/policies.html

```diff
@@ -0,0 +1,51 @@
+<div class="container">
+
+  <h3>License</h3>
+
+  <p>OmegaT is distributed under the GPL v3 license agreement.</p>
+
+  <p>Basically, that means that you have:</p>
+  <ul>
+    <li>the freedom to use the software for any purpose,</li>
+    <li>the freedom to change the software to suit your needs,</li>
+    <li>the freedom to share the software with your friends and neighbors, and</li>
+    <li>the freedom to share the changes you make.</li>
+  </ul>
+
+  <p>These 4 items are a good summary but you can also read the <a
+      href="https://sourceforge.net/p/omegat/code/ci/master/tree/release/OmegaT-license.txt">whole license</a>.</p>
+
+  <h3>Privacy Policy</h3>
+
+  <p>OmegaT does not collect any user information and does not transmit any information without the explicit consent
+    of the user.</p>
+
+  <p>The user or their project manager can set OmegaT to use online services. In that case, OmegaT shares information
+    with online services:</p>
+
+  <ul>
+    <li>Access to machine translation services</li>
+    <li>Access to team projects</li>
+  </ul>
+
+  <p>OmegaT automatically checks the availability of newer version on its development server but the user can easily opt
+    out in the Preferences settings and does not transmit any user information.</p>
```

Comment:
```suggestion
    out in the Preferences settings.</p>
```

Reply:
Apologies. I missed a glitch and a rewording. We should be good now. Thank you very much.

Reply:
Thank you! 
I'd rather take a bit longer and not leave typos and a public facing page :)

Reply:
Et joyeux Noël. On est déjà le 25 ici... :-)

Reply:
Joyeux Noël!